stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/EditReactionsView.kt

```diff
@@ -0,0 +1,87 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.Reaction
+import io.getstream.chat.android.ui.utils.UiUtils
+
+public class EditReactionsView : ReactionsView {
+    private val bubblePaint = Paint().apply {
+        style = Paint.Style.FILL
+        color = Color.WHITE
+    }
+
+    public constructor(context: Context) : super(context) {
+        init(context, null)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init(context, attrs)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+        init(context, attrs)
+    }
+
+    private fun init(context: Context, attrs: AttributeSet?) {
+        setStyle(EditReactionsViewStyle(context, attrs))
+    }
+
+    override fun createReactionItems(message: Message, isMyMessage: Boolean): List<ReactionsAdapter.ReactionItem> {
+        return UiUtils.getReactionTypes().keys.map { reactionType ->
+            message.ownReactions
+                .any { it.type == reactionType }
+                .let { ReactionsAdapter.ReactionItem(Reaction(type = reactionType), it) }
+        }
+    }
+
+    override fun drawReactionsBubble(canvas: Canvas, isMyMessage: Boolean) {
+        drawBubbleRoundRect(canvas)
+        drawTailBiggerCircle(canvas, isMyMessage)
+        drawTailSmallerCircle(canvas, isMyMessage)
+    }
+
+    private fun drawBubbleRoundRect(canvas: Canvas) {
+        canvas.drawRoundRect(
+            0f,
+            0f,
+            width.toFloat(),
+            reactionsViewStyle.reactionsBubbleHeight.toFloat(),
+            reactionsViewStyle.reactionsBubbleRadius.toFloat(),
+            reactionsViewStyle.reactionsBubbleRadius.toFloat(),
+            bubblePaint
+        )
+    }
+
+    private fun drawTailBiggerCircle(canvas: Canvas, isMyMessage: Boolean) {
+        val offset = reactionsViewStyle.reactionsBigTailCircleOffset.toFloat().let {
+            if (isMyMessage) it else -it
```

Comment:
We shouldn't use `isMyMessage` to decide if the reaction bubble go right or left.
We should try to use left/right or start/end instead. Some customers already asked us to align all messages (mine and their) on the same side, and on that case we will get some issue or be confused when render this view

Reply:
Did I get it correctly, bubble style (solid grey / white with outline) is still supposed to depend on message affiliation (mine or theirs), we just need an independent property to control the `mirroring` of the bubble?

Reply:
Yep, we will need two style properties there, one for bubble background and another for the message alignment